papermill/translators.py

```diff
@@ -377,6 +547,8 @@ def assign(cls, name, str_val):
 papermill_translators.register("matlab", MatlabTranslator)
 papermill_translators.register(".net-csharp", CSharpTranslator)
 papermill_translators.register(".net-fsharp", FSharpTranslator)
+papermill_translators.register(".net-powershell", PowershellTranslator)
+papermill_translators.register("pysparkkernel", PythonTranslator)
```

Comment:
What about the `sparkkernel` and SparkK kerenels?

Reply:
gp - registered